openbb_platform/core/tests/app/logs/test_logging_service.py

```diff
@@ -84,11 +84,10 @@ def test_correctly_initialized():
         "openbb_core.app.logs.logging_service.LoggingService._log_startup",
         mock_log_startup,
     ):
-        LoggingService(
+        ls = LoggingService(  # noqa: F841  # pylint: disable=unused-variable
```

Comment:
If it's really unused there is a convention for this in python to name the variable _

Try and you will not need the comment that silences the linter

Reply:
That'll do, ya.